### ISSUE TITLE
Fabric mod did not start on 1.20.6

### ISSUE THREAD

**Issue by nisutec** [enjarai/show-me-your-skin#77]
Hello,

I have installed the following mods.

-	show-me-your-skin-1.10.0+1.20.5
-	cicada-lib-0.7.2+1.20.5-and-above
-	fabric-api-0.98.0+1.20.6

But the Fabric Loader crashes with the following message when I start the game.

```
Running Fabric Loader version 0.15.11.
Some of your mods are incompatible with the game
or each other!
A potential solution has been determined, this may
resolve your problem:
- Install cardinal-components-base, any
version.
- Install cardinal-components-entity, any
version.
More details:
- Mod "Show Me Your Skin!' (shoumesourskin)
1.100+1.205 requires any version of
cardinal-components-base, which is missing!
- Mod 'Show Me Your Skin! (shoumeyourskin)
1.100+1.20.5 requires any version of
cardinal-components-entity, which is missing!
```

Is there any additional requirement to run the mod on 1.20.6.

**Comment by enjarai** [comment]
Cardinal Components API is a dependency that's supposed to be included with the mod jar... I'll have to see why that's not working properly. For now you can download it manually: https://modrinth.com/mod/cardinal-components-api

**Comment by enjarai** [comment]
Fixed in 1.10.1.
